paper-server/src/main/java/io/papermc/paper/util/ItemComponentSanitizer.java

```diff
@@ -66,11 +66,21 @@ private static ItemEnchantments dummyEnchantments() {
     public static int sanitizeCount(final ItemObfuscationSession obfuscationSession, final ItemStack itemStack, final int count) {
         if (obfuscationSession.obfuscationLevel() != ItemObfuscationSession.ObfuscationLevel.ALL) return count; // Ignore if we are not obfuscating
 
-        if (GlobalConfiguration.get().anticheat.obfuscation.items.binding.getAssetObfuscation(itemStack).sanitizeCount()) {
-            return 1;
-        } else {
+        if (!GlobalConfiguration.get().anticheat.obfuscation.items.binding.getAssetObfuscation(itemStack).sanitizeCount() || count <= 1) {
```

Comment:
Maybe first check count? Seems like a free performance gain for most equipment

Reply:
In reality, it most likely does not matter that much and honestly, branch prediction taking the wrong path might be worse.

Reply:
It's indeed not worth big discussion as the gain is minimal, so won't continue past this message explaining my thoughts. If someone could mark this as resolved would be glad 

Since the check is already present, it makes sense to make the easy check first that already has all the data on stack instead of the pointer chasing through all the ram, especially since in current implementation the majority of items passing here will be from entiy equipment, which is mostly either 1 for unstackable items (armor) or 0 for empty slots, so since it's true it's not gonna even need to check the config